_stbt/imgutils.py

```diff
@@ -0,0 +1,164 @@
+import inspect
+import os
+from collections import namedtuple
+
+import cv2
+import numpy
+
+from .logging import ddebug
+from .types import Region
+
+
+class Frame(numpy.ndarray):
```

Comment:
Doesn't it make more sense for `Frame` to go in `_stbt.types`?

Reply:
I see, it's because `types.py` isn't supposed to have any dependencies outside the Python standard library. We could make an exception for numpy?

Reply:
I would say leave it be for now and we can revisit the decision in the future.